### PR TITLE
Move language selector to footer

### PR DESCRIPTION
**Pull request by mirjamverloop**

### CODE REVIEW

File: components/generic/LanguageSelector.vue [hunk 1]
```diff
@@ -47,10 +45,7 @@
 </script>
 
 <style lang="scss" scoped>
-  @import "./assets/scss/variables.scss";
-
-  .dropdown {
-    margin-top: 20px;
+  /*.dropdown {
```
Comment (monkemedia) [inline components/generic/LanguageSelector.vue]: could this now be removed?

Comment (mirjamverloop) [reply components/generic/LanguageSelector.vue]: Yes, you are right, thanks!

File: tests/unit/components/PageFooter.spec.js [hunk 2]
```diff
@@ -34,4 +34,15 @@ describe('components/search/PageFooter', () => {
...
+  it('contains the language selector', () => {
+    const wrapper = factory();
+    wrapper.setProps({
+      enableLanguageSelector: true
+    });
+    const selector = wrapper.find('[data-qa="language selector"]');
+
+    selector.isVisible().should.equal(true);
+  });
+
```
Comment (lbiedinger) [inline tests/unit/components/PageFooter.spec.js]: this raises a linting error at the start of the unit tests
```
 Module Warning (from ./node_modules/@vue/cli-plugin-eslint/node_modules/eslint-loader/index.js):
  error: Trailing spaces not allowed (no-trailing-spaces) at tests/unit/components/PageFooter.spec.js:47:1:
    45 |     selector.isVisible().should.equal(true);
    46 |   });
  > 47 |   
       | ^
    48 | });
    49 | 

```

Comment (mirjamverloop) [reply tests/unit/components/PageFooter.spec.js]: That has been removed, @lbiedinger

**Pull request merged**